libdevcore/Common.h

```diff
@@ -135,6 +135,16 @@ inline u256 s2u(s256 _u)
 		return u256(c_end + _u);
 }
 
+inline std::ostream& operator<<(std::ostream& os, bytes const& _bytes)
```

Comment:
I'm not sure if this is the most useful representation. What about implementing this as
``os << toHex(_bytes)``?